eslint.config.js

```diff
@@ -41,7 +40,7 @@ export default [
 		files: ["**/*.json"],
 		ignores: ["**/package-lock.json"],
 		language: "json/json",
-		...json.configs.recommended,
+		extends: [json.configs.recommended],
```

Comment:
Let's use the string version here.
```suggestion
		extends: ["json/recommended"],
```

Reply:
```js
import json from "./src/index.js";
```

Unfortunately, `json` is imported directly from `./src/index.js`, it causes an error when running ESLint, as shown below:

```txt
TypeError: Plugin "json" not found.
    at findPluginConfig (path)
    at path
    at Array.map (<anonymous>)
    at processExtends (path)
    at path
    at Array.flatMap (<anonymous>)
    at processConfigList (path)
    at defineConfig (path)
    at path
    at ModuleJob.run (node:internal/modules/esm/module_job:234:25)

```

Would it be alright to leave it as it is?

Reply:
Sorry, you'd also need to include:

```js
{
    plugins: { json }
}
```

Reply:
Thank you!

I've added a new commit https://github.com/eslint/json/pull/93/commits/5da7bcb4bbdd720f6a62f455af764f21a19ace00